pkgs/tools/misc/plantuml/plantuml-c4.nix

```diff
@@ -12,8 +12,8 @@
 
 let
   c4-lib = fetchzip {
-    url = "https://github.com/plantuml-stdlib/C4-PlantUML/archive/88a3f99150c6ff7953c4a99b184d03412ffdedb1.zip";
-    sha256 = "sha256-vk4YWdGb47OsI9mApGTQ7OfELRZdBouzKfUZq3kchcM=";
+    url = "https://github.com/plantuml-stdlib/C4-PlantUML/archive/refs/tags/v2.8.0.zip";
+    sha256 = "sha256-pGtTFg7HcAFYPrjd+CAaxS4C6Cqaj94aq45v3NpiAxM=";
```

Comment:
```suggestion
    hash = "sha256-pGtTFg7HcAFYPrjd+CAaxS4C6Cqaj94aq45v3NpiAxM=";
```

Reply:
Thanks for the hint, changed the second occurrence as well: https://github.com/NixOS/nixpkgs/commit/0061010fbc50cff2a9c9c0abe3b7de03093806e8#diff-9de1fe7c42caf79351ddb7ada270e1d0a82c30535f6836e507e1ea1f3c134b94R21